ddout/ddout.c

```diff
@@ -348,6 +350,11 @@ main()
 	}
       }
       if(fd >= 0) {
+        if(flock(fd,LOCK_EX|LOCK_NB)) {
```

Comment:
I notice the man page says flock doesn't work over NFS. I wonder if this will be noisy in that case. At least it's benign though.

I see fcntl (POSIX) locks work over NFS, but the API is a bit more involved.

Reply:
Not an issue anymore I guess.